base/applications/rapps/settingsdlg.cpp

```diff
@@ -58,7 +58,7 @@ IsUrlValid(const WCHAR *Url)
     UrlComponmentInfo.dwStructSize = sizeof(UrlComponmentInfo);
     UrlComponmentInfo.dwSchemeLength = 1;
 
-    BOOL bSuccess = InternetCrackUrlW(Url, wcslen(Url), 0, &UrlComponmentInfo);
+    BOOL bSuccess = InternetCrackUrlW(Url, (UINT)wcslen(Url), 0, &UrlComponmentInfo);
```

Comment:
```suggestion
    BOOL bSuccess = InternetCrackUrlW(Url, DWORD(wcslen(Url)), 0, &UrlComponmentInfo);
```
to stay with the actual parameter type from the header/official documentation, and use the C++ style cast as you use elsewhere.

Side-question: Shouldn't we count the NUL-terminator there too (i.e. `wcslen(Url) + 1`) ?